pwnlib/log.py

```diff
@@ -427,15 +427,15 @@ def warn(self, *args, **kwargs):
         """Alias for :meth:`warning`."""
         return self.warning(*args, **kwargs)
 
-    def error(self, message, *args, **kwargs):
+    def error(self, message, *args, original_exc=None, **kwargs):
         """error(message, *args, **kwargs)
 
         To be called outside an exception handler.
```

Comment:
Based on this comment, maybe we move the log.error out of except block:
https://github.com/Gallopsled/pwntools/blob/e841b500d7b6a4b8ecac06b7e5875badb7622b80/pwnlib/tubes/remote.py#L89